### DISCUSSION TITLE
mspider/user-guide/service-mesh/capacity-resources

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3707] [Announcements]
# mspider/user-guide/service-mesh/capacity-resources

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/service-mesh/capacity-resources.html

**Comment by nicole-lihui** [comment]
新增了资源规模,需要同步文档
